src/Front-end/Messages.js

```diff
@@ -1,70 +1,204 @@
 import "./Messages.css"
+import React, { useState, useRef, useEffect } from "react";
+import ReactDOM from 'react-dom'
 import NavBar from "./NavBar";
-
 import { pageTheme } from "./util";
+import { Typography, Box, TextareaAutosize ,Container, Grid, Paper, TextField, ThemeProvider, List, ListItem, Checkbox, ListItemText, Slider, Input, MenuItem, Select, Stack, Button } from '@mui/material';
+import axios from 'axios';
+import { auth } from "../login/loginconfig";
+
+
+// }
+// const handleNewChat = () =>{
+//     axios.newChat('http://localhost:8000/api/messages',data).then(res => {
+//         // clear the form fields
+//         // notify user that their listing was posted
+//       }).catch(error => {
+//         console.error('Error posting data:', error);
+// })}
+
+//sample messages only sent by the current user
+
+//param to pass in through axios, need component to operate with verified email address
+export const USERNAME = "bruh1"
```

Comment:
Should we be doing `const USERNAME = auth.currentUser ? auth.currentUser.email : '';` instead? Also, is there a reason this and USERLIST are exported?

Reply:
We should include the const USERNAME = auth.currentUser ? auth.currentUser.email : ''; , however I need a way to get usernames of those the user wants to start a new chat with. Maybe the message seller button on the listings page can somehow parse the users email?

Reply:
That would be super doable! If you look at the code for the listing page, we display the seller's ID. That same ID could be passed as a parameter when running the messages page code. Does th